src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -256,11 +256,14 @@ export class ConnectionWidget extends lifecycle.Disposable {
 	protected addCustomConnectionOptions(): void {
 		if (this._customOptions.length > 0) {
 			this._customOptionWidgets = [];
+			let trueInputValue = localize('boolean.true', 'True');
```

Comment:
Put these at the module level (or a common constants file) so we don't have to call localize every single time for constant strings